people-names/words.txt

```diff
@@ -6,3 +6,4 @@ Koide
 Menegatti
 Miura
 Miyake
+VRichard
```

Comment:
Sorry, I meant you should fix here instead of adding the word.
https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_velocity_planner/src/node.cpp#L324

Reply:
@kenji-miyake The error is fixed after modifying source comment.
So, I close this pull-request